server/scripts/import-tests/index.js

```diff
@@ -465,6 +466,33 @@ const getTests = ({
                         tokenizedAssertionStatement ||
                         assertion.assertionStatement;
                     result.assertionPhrase = assertion.assertionPhrase;
+                    result.assertionExceptions = data.commands.flatMap(
+                        command => {
+                            return command.assertionExceptions
+                                .filter(
+                                    exception =>
+                                        exception.assertionId ===
+                                        assertion.assertionId
+                                )
+                                .map(({ priority: assertionPriority }) => {
+                                    let priority = '';
+                                    if (assertionPriority === 0)
+                                        priority = 'EXCLUDE';
+                                    if (assertionPriority === 1)
+                                        priority = 'MUST';
+                                    if (assertionPriority === 2)
+                                        priority = 'SHOULD';
+                                    if (assertionPriority === 3)
+                                        priority = 'MAY';
+
```

Comment:
It would be nice to have a utility variable that maps these ints to these strings (which could also be enums). This would help centralize and lift the significance of these relationships which would be helpful for someone like me that was just editing `assertionExceptions`` on my local aria-at without fully understanding what the numbers meant.

Reply:
Good point. There's the `server/resolvers/helpers/convertAssertionPriority.js` file, which could be moved to a more appropriate location, and be expanded to something like:

```javascript
const convertAssertionPriority = priority => {
    if (priority === 0) return 'EXCLUDE';
    if (priority === 'REQUIRED' || priority === 1) return 'MUST';
    if (priority === 'OPTIONAL' || priority === 2) return 'SHOULD';
    if (priority === 3) return 'MAY';
};

module.exports = convertAssertionPriority;
```

Would this support what you're thinking of?

Reply:
That would work. Thank you!

Reply:
Done in https://github.com/w3c/aria-at-app/pull/863/commits/3d1fbcb61e4e42ed3cc56802fe90827f7a001d8a